lib/containers/k8s.pm

```diff
@@ -84,7 +84,9 @@ sub install_kubectl {
             # kubectl is in the container module
             add_suseconnect_product(get_addon_fullname('contm'));
             # SLES-15SP2+ ships a specific kubernetes client version. Older versions hold a version-independent kubernetes-client package.
-            if (is_sle(">15-SP1")) {
+            if (is_sle(">=15-SP5")) {
```

Comment:
```suggestion
            if (is_sle("==15-SP5")) {
```

I think it will more clear message , also I would assume for SP6 we would need some code tweaking here anyway ...

Reply:
This would lead to a syntax error. Anyway, I believe we should try to use zypper's capabilities to download the best fitting package, rather than bumping the versions for each service pack.

Reply:
I have just tested the installation in TW, `kubernetes-client` always pulled the newest one `1.24`. Do you remember why we have introduced this complex logic?

Reply:
I **THINK** (!!!!) because at some point `kubernetes-client` stopped working.

Reply:
@mloviska [SLE 15-SP4](https://pdostal-server.suse.cz/tests/3464):
```
# zypper -n in kubernetes-client | cat; ( exit ${PIPESTATUS[0]} ); echo s_CXw-$?-
Refreshing service 'Basesystem_Module_15_SP4_x86_64'.
Refreshing service 'Containers_Module_15_SP4_x86_64'.
Refreshing service 'Desktop_Applications_Module_15_SP4_x86_64'.
Refreshing service 'Development_Tools_Module_15_SP4_x86_64'.
Refreshing service 'Legacy_Module_15_SP4_x86_64'.
Refreshing service 'Python_3_Module_15_SP4_x86_64'.
Refreshing service 'SUSE_Linux_Enterprise_Server_15_SP4_x86_64'.
Refreshing service 'SUSE_Linux_Enterprise_Workstation_Extension_15_SP4_x86_64'.
Refreshing service 'Server_Applications_Module_15_SP4_x86_64'.
Refreshing service 'Web_and_Scripting_Module_15_SP4_x86_64'.
Building repository 'SLE-Module-Containers15-SP4-Pool' cache [....done]
Retrieving repository 'SLE-Module-Containers15-SP4-Updates' metadata [.done]
Building repository 'SLE-Module-Containers15-SP4-Updates' cache [....done]
Loading repository data...
Reading installed packages...
'kubernetes-client' not found in package names. Trying capabilities.
No provider of 'kubernetes-client' found.
```